Bottom: e42db9ab7446a6ad4e7927885af8ce3b27671bfc
Top:    12f28d14dd17d7d75cb06424e8fe27bb5fb3f984
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-29 15:22:27 +0800

Refresh of vhost_net-tx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 20614eb..ebc0c53 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -578,7 +578,7 @@ static void handle_tx(struct vhost_net *net)
 				pr_debug("Truncated TX packet: "
 					" len %d != %zd\n", err, len);
 			if (!zcopy) {
-				vhost_update_used_idx(vq, 1);
+				vhost_add_used_idx(vq, 1);
 				vhost_signal(&net->dev, vq);
 			} else if (!zcopy_used) {
 				vhost_add_used_and_signal(&net->dev,
